templates/webapps/galaxy/workflow/run.mako

```diff
@@ -576,15 +576,14 @@ if wf_parms:
 
 %if has_upgrade_messages:
 <div class="warningmessage">
-    Problems were encountered when loading this workflow, likely due to tool
-    version changes. Missing parameter values have been replaced with default.
-    Please review the parameter values below.
+    Warning: Some tools in this workflow have been updated. The workflow may still work, but any new inputs will have default values.
```

Comment:
I think parameter is a pretty programmer centric term... and is unfamilar to the bench folks I work with.  Is there a better term than "input" to describe the information being specified to a tool?